java/ingest/ingest-core/src/main/java/sleeper/ingest/job/status/IngestJobStartedEvent.java

```diff
@@ -44,17 +47,54 @@ public static Builder builder() {
         return new Builder();
     }
 
+    /**
+     * Creates an instance of this class. This constructor is specifically for ingest jobs and creates an event that
+     * marks the start of a job run. This is not used for bulk import jobs, as they have a validation event before
+     * this, and this validation event marks the start of a job run. Bulk import jobs should use the
+     * {@link IngestJobStartedEvent#validatedIngestJobStarted} constructor.
+     *
+     * @param  taskId    the task ID
+     * @param  job       the ingest job
+     * @param  startTime the start time
+     * @return           an instance of this class
+     */
     public static IngestJobStartedEvent ingestJobStarted(String taskId, IngestJob job, Instant startTime) {
         return ingestJobStarted(job, startTime).taskId(taskId).build();
     }
 
+    /**
+     * Creates an instance of this class. This constructor is specifically for ingest jobs and creates an event that
+     * marks the start of a job run.
+     * <p>
+     * This is not used for bulk import jobs, as they have a validation event before
+     * this, and this validation event marks the start of a job run. Bulk import jobs should use the
+     * {@link IngestJobStartedEvent#validatedIngestJobStarted} constructor.
+     *
+     * @param  job       the ingest job
+     * @param  startTime the start time
+     * @return           an instance of this class
+     */
     public static Builder ingestJobStarted(IngestJob job, Instant startTime) {
         return builder()
                 .job(job)
                 .startTime(startTime)
                 .startOfRun(true);
     }
 
+    /**
+     * Creates an instance of this class. This constructor is specifically for bulk import jobs and creates an event
+     * that indicates the job has been picked up in the Spark cluster by the driver.
+     * <p>
+     * Note that this does not mark the start of a job run. Once the bulk import starter picks up a bulk import job, it
+     * validates the job and saves an event then, which marks the start of a job run.
+     * <p>
+     * This is not used for ingest jobs this, and this validation event marks the start of a job run. Bulk import jobs
+     * should use the {@link IngestJobStartedEvent#ingestJobStarted} constructor.
```

Comment:
The last paragraph needs fixing here.

Reply:
This isn't a validation event, that's a separate event. It's the validation event that marks the start of the run, but that's already been said in the previous paragraph. There's also a typo at "ingest jobs this". Should this paragraph just explain that ingest jobs use the other constructor?